### ISSUE TITLE
请问BN层中momentum这个为什么是分数衰减的

### ISSUE THREAD

**Issue by Lutong-Qin** [taoyang1122/MutualNet#12]
尊敬的作者您好，感谢您出色的工作。
1.我想问您一下，BN层的计算中，mumentum为什么是1,1/2,1/3,1/4,1/5,1/6,1/7这样子衰减呢，是这样衰减比 mumentum 设置为一个常熟(比如0.1)的效果要好么？谢谢您的帮助，非常感谢。
2.cumulutive running average ：我没查到这个相关的移动平均，请问这个出自哪里。
不胜感激。
![image](https://user-images.githubusercontent.com/51037385/206891448-ed7ec85e-c540-435a-8651-e07ed3b552fb.png)

**Comment by taoyang1122** [comment]
Thanks for your interest. We set the BN momentum to `1 / (batch_idx)` to make the running mean and running var as the simple average of all batches. You may refer to the doc of [BN Layers](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html) to see how it works.

**Comment by Lutong-Qin** [comment]
Thank you very much for solving my problem. I am sorry that I forgot to reply to your message. I am very sorry.

**Comment by taoyang1122** [comment]
No problem. I am closing the issue.
